modules/cnf-du-pod-mutation.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/cnf-provisioning-and-deploying-a-distributed-unit.adoc
+
+[id="cnf-du-pod-mutation_{context}"]
+
+= Pod mutation
```

Comment:
While I admire the austerity of this short title, please add "workload partitioning" into it.